### ISSUE TITLE
custom paths for components/ui

### ISSUE THREAD

**Issue by xmlking** [huntabyte/shadcn-svelte#547]
### Describe the feature in detail (code, mocks, or screenshots encouraged)

I am using monorepo and want to control where components get added with `cli add`  command.
this still don't allow changing path  for `components:ui` type
```
npx shadcn-svelte@latest add -p src/lib/components/shadcn button
```

Ask: let the users customized where components get added via either CLI args or via  `components.json`

### What type of pull request would this be?

None

### Provide relevant links or additional information.

```
apps/myapp <-- `sveltekit app`
libs/ui/ <-- `shared components lib`
        src/lib/components/shadcn
               button
               dropdown-menu
        components.json
```

**Comment by Skylli202** [comment]
@huntabyte What's your thought on this issue? I think I could take a look at how to solve this.

The CLI could just ask the path, like when the `init` command ask for the global CSS file. But that would also add an extra step for everyone who is not using such mono-repo (which is the majority of people, no?). 

@xmlking It might be a bit dirty, but have you try to move the `components.json` at the root or your mono-repo? Update the configuration stored in it accordingly? It should do the job.

**Comment by xmlking** [comment]
For reference, Here is my monorepo  https://github.com/xmlking/spectacular
Shadcn components are at packages/ui with components.json in it.
Consuming  app is at apps/console-sc

**Comment by huntabyte** [comment]
Added in `@next`

**Comment by Oudwins** [comment]
Hey! I too I'm having trouble with a monorepo. But I see that you might have fixed the issue in your repo @xmlking 

The issue I am having is with the components.json file and the aliases. If I set the aliases like they are meant to be set (i.e "$lib/components" for example) they evaluate to the application that is consuming the ui package. And if I set the them to relative paths the CLI throws a " - Import aliases must use existing path aliases defined in your tsconfig.json (e.g. "utils": "$lib/utils")." error.

How did you do it?
